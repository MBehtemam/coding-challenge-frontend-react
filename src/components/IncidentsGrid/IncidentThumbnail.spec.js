import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import IncidentThumbnail from './IncidentThumbnail';

describe('Test suits for IncidentsThumbnail', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<IncidentThumbnail />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
