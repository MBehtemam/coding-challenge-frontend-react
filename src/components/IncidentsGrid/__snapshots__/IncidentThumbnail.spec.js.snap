// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for IncidentsThumbnail should match with snapshot 1`] = `
.c0 {
  height: 100px;
  width: 100px;
}

<img
  className="c0"
/>
`;
